main.go

```diff
@@ -2,61 +2,31 @@ package main
 
 import (
 	"./levels"
+	"flag"
+	"fmt"
+	"os"
 )
 
 func main() {
-	challenge := levels.NewChallenge("first")
-	challenge.AddLevel(levels.Level{
-		TestCmd: `true`,
-		Text: `
-# First alternative bash excercise
+	print_flag := flag.Bool("print", false, "print loaded levels and exit")
+	flag.Parse()
 
-Hi there.
-
-I understand that a command line talking to you might not make sense right now
-but bear with me, things *will* get better.
-
-First of all, you need to learn how to move from one directory to another. This
-can be done using the "cd" commad which is a very short abbreviation for
-**'change directory'**.  For instance if I wanted to go to the /usr directory
-I would type
-
-	$ cd /usr
-
-Got it?
-
-I am pretty sure you do. Now use this cd comand and go to the /tmp directory
-`,
-	})
-
-	challenge.AddLevel(levels.Level{
-		Name:    `l01`,
-		TestCmd: `test $(pwd) = "/tmp"`,
-		Text: `
-I see you made it, awesome !
+	if len(flag.Args()) < 1 {
+		fmt.Printf("\n\nNo input file\n\n")
+		fmt.Printf("usage: %s path\n\n", flag.Args()[0])
```

Comment:
@mrshu When no argument is specified this line throws `panic: runtime error: index out of range`. I think flag doesn't have have 0th element name of the file, which would make sense since you are checking if a length of `flag.Args()` is 0.